### DISCUSSION TITLE
Anyone could help with GB?

### DISCUSSION THREAD

**Discussion by maciekpastuszak** [drknzz/GitHub-Achievements#131] [Q&A] [answered]
Anyone?

**Comment by muller-g** [comment]
Hi, 🧠

**Reply by maciekpastuszak** [reply]
Cool. Can you set one for me?

**Comment by SaifuddinGsohan** [comment]
Of course! I'd be happy to help. However, I need a bit more context to understand what you mean by "GB." GB could stand for many things, such as gigabytes (a unit of digital storage), Great Britain (a country), or something else entirely. Could you please provide more details or clarify your question so I can assist you better?

**Comment by ashishkumar30** [comment]
yes

**Reply by maciekpastuszak** [reply]
Can you set one up for me? I will mark yours

**Comment by ogprzemo** [comment]
hello

**Reply by ogprzemo** [reply]
can you answer me?

**Reply by maciekpastuszak** [reply]
Yes. Mark my answer and I will mark yours

**Comment by paciekFly** [comment, marked as answer]
yes
